pkg/api/norman/customization/cluster/validator.go

```diff
@@ -332,6 +332,171 @@ func (v *Validator) validateGenericEngineConfig(request *types.APIContext, spec
 
 }
 
+func (v *Validator) validateAKSConfig(request *types.APIContext, cluster map[string]interface{}, clusterSpec *v32.ClusterSpec) error {
```

Comment:
I don't see where this is used, is it supposed to be here https://github.com/rancher/rancher/blob/5296575612b743a658abf0b8a78f27c553db1d5d/pkg/api/norman/customization/cluster/validator.go#L74 ?

Reply:
Thnak you @cmurphy, I missed that :)